src/main/webapp/WEB-INF/view/login.jsp

```diff
@@ -13,36 +13,44 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 --%>
+
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Login</title>
-  <link rel="stylesheet" href="/css/main.css">
+  <title>CodeU Chat App</title>
+  <link rel="stylesheet" href="/css/loginpage.css">
 </head>
 <body>
 
-  <%@ include file="../component/navbar.jsp" %>
+  <nav>
+    <div class="nav-wrapper">
+        <b id="navTitle" href="/ center">CODEU CHAT APP</b>
+    </div>
+  </nav>
 
   <div id="container">
-    <h1>Login</h1>
-
-    <% if(request.getAttribute("error") != null){ %>
-        <h2 style="color:red"><%= request.getAttribute("error") %></h2>
-    <% } %>
-
-    <form action="/login" method="POST">
-      <label for="username">Username: </label>
-      <br/>
-      <input type="text" name="username" id="username">
-      <br/>
-      <label for="password">Password: </label>
-      <br/>
-      <input type="password" name="password" id="password">
-      <br/><br/>
-      <button type="submit">Login</button>
-    </form>
-
-    <p>New users can register <a href="/register">here</a>.</p>
+    <div
+      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
```

Comment:
It's usually better to keep css in the css part rather than using inline style. The motivation is to keep the same logic in the same place.